assets/js/blocks/cart-checkout/mini-cart/style.scss

```
@@ -100,16 +100,25 @@ h2.wc-block-mini-cart__title {
 	flex-direction: column;
 	flex-grow: 1;
 	overflow-y: hidden;
-	padding: 0 $gap;
+	padding: $gap $gap 0;
 
 	.wc-block-mini-cart__products-table {
 		margin-bottom: auto;
 		margin-right: -$gap;
 		overflow-y: auto;
 		padding-right: $gap;
 
```

Is there a particular reason for adding this style?

I just tried with this and it seems to work

```suggestion
.wc-block-cart-items__row::after {
    border: none !important
}
```

---

Thanks for the review!

The additional code adds a bit of space on top of the mini cart contents further improving it visually.

---

Ok! It makes sense!